tests/conftest.py

```diff
@@ -75,6 +75,9 @@ def pytest_addoption(parser):
     parser.addoption("--neighbor_type", action="store", default="eos", type=str, choices=["eos", "sonic"],
                     help="Neighbor devices type")
 
+    # FWUtil options
+    parser.addoption('--fw-pkg', action='store', help='Firmware package file')
```

Comment:
@alexrallen why do we need to have this option in common conftest?

Reply:
I was having significant issues getting pytest to properly integrate the new option when I added it to the fwutil specific conftest.py I am open to any suggestions here but I found some documentation stating that pytest had some bugs with regard to adding options in nested conftest files and figured that was sufficient justification to add it here. 

https://docs.pytest.org/en/latest/reference/reference.html#_pytest.hookspec.pytest_addoption
>This function should be implemented only in plugins or conftest.py files situated at the tests root directory due to how pytest discovers plugins during startup.

We can also see there are some other options here so I think this is not unprecedented.